### DISCUSSION TITLE
请教下各位怎么播放云盘上传的ape文件，获取到了链接，但是用web audio api没法播放呢？

### DISCUSSION THREAD

**Discussion by wood3n** [Binaryify/NeteaseCloudMusicApi#1755] [Q&A] [answered]
<img width="1483" alt="image" src="https://github.com/Binaryify/NeteaseCloudMusicApi/assets/31716713/864b6184-ce16-40ba-98dc-f64ed3366782">

**Comment by Binaryify** [comment, marked as answer]
web 没法解析吧

**Reply by wood3n** [reply]
应该是，我看web audio api不支持 😢

**Comment by wood3n** [comment]
暂时找了个方法，用`@ffmpeg/ffmpeg`在浏览器里直接将 ape 转成 mp3 然后播放 🤣
